Tribler/Main/tribler_main.py

```diff
@@ -8,7 +8,9 @@
 #
 # see LICENSE.txt for license information
 #
+from Tribler.Policies.BoostingManager import BoostingManager
```

Comment:
Pylint: `13: C0411: (wrong-import-order), : standard import "import shutil" comes before "from Tribler.Policies.BoostingManager import BoostingManager"`. This import should go down.